homeassistant/components/enocean/cover.py

```diff
@@ -0,0 +1,276 @@
+"""Support for EnOcean roller shutters."""
+from __future__ import annotations
+
+import asyncio
+from enum import Enum
+import logging
+
+from enocean.protocol.constants import RORG
+from enocean.protocol.packet import RadioPacket
+from enocean.utils import combine_hex
+import voluptuous as vol
+
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    DEVICE_CLASSES_SCHEMA,
+    PLATFORM_SCHEMA,
+    CoverDeviceClass,
+    CoverEntity,
+    CoverEntityFeature,
+)
+from homeassistant.const import CONF_DEVICE_CLASS, CONF_ID, CONF_NAME
+from homeassistant.core import HomeAssistant
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.dispatcher import dispatcher_send
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
+
+from .const import SIGNAL_SEND_MESSAGE
+from .device import EnOceanEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = "EnOcean roller shutter"
+
+CONF_SENDER_ID = "sender_id"
+
+WATCHDOG_TIMEOUT = 1
+WATCHDOG_INTERVAL = 0.2
```

Comment:
The interval can be the same as the timeout I think. The only thing gained by having a shorter interval is that the watchdog task exits faster. That doesn't seem to matter much though. What do you think?

Reply:
I'll test that in my local setup - in theory this should work.

Reply:
As expected: it works. I've adjusted the interval, see https://github.com/home-assistant/core/pull/75356/commits/2306d243b29b8ff9bab6bd203ea18616edc4fde2.